data/caco2_wang/meta.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: caco2_wang
+description: |-
+    The human colon epithelial cancer cell line, Caco-2, is used as an in vitro model
+            to simulate the human intestinal tissue. The experimental result on the rate of drug passing through
+            the Caco-2 cells can approximate the rate at which the drug permeates through the human intestinal tissue.
+targets:
+    - id: permeability
+      description: Caco-2 cell effective permeability.
+      units: logPapp
+      type: continuous
+      names:
+          - Caco-2 cell effective permeability
+          - Caco-2 cell permeability
+          - Caco-2 permeability
+          - permeability
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: compound_id
+      type: Other
+      description: Compound id / name
+      names:
+          - compound id
+          - compound name
+license: CC BY 4.0
+links:
+    - url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
+      description: tdcommons.ai dataset url
+    - url: https://pubs.acs.org/doi/10.1021/acs.jcim.5b00642
+      description: corresponding publication
+num_points: 910
+url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
```

Comment:
```suggestion
url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
```

Reply:
It seems URL is duplicated. My fault as I didn't fix the check for this yet

Reply:
thanks for catching this, I removed one! :-)